support/jwt/src/main/kotlin/com/xorker/xpr/support/jwt/JwtSecretKey.kt

```diff
@@ -0,0 +1,18 @@
+package com.xorker.xpr.support.jwt
+
+import io.jsonwebtoken.security.Keys
+import java.security.Key
+import org.springframework.beans.factory.annotation.Value
+
+object JwtSecretKey : SignatureKey {
+    @Value("\${jwt.secret}")
+    private lateinit var secret: String
+    private var secretKey: Key? = null
+
+    override fun getKey(): Key {
+        if (secretKey == null) {
+            secretKey = Keys.hmacShaKeyFor(secret.toByteArray())
+        }
+        return secretKey!!
+    }
```

Comment:
p1. 가급적 notnull로 할 수 있는 방법을 제안드려봅니다.

이에 맞게 interface도 변경 하면 될 것 같아요

```suggestion
    override val key: Key by lazy {
            return Keys.hmacShaKeyFor(secret.toByteArray())
    }
```